code/game/modifiers/modifiers_chem.dm

```diff
@@ -82,4 +82,30 @@
 	..()
 	if (isliving(target))
 		var/mob/living/L = target
-		L.set_light(0)
\ No newline at end of file
+		L.set_light(0)
+
+//Doubleburn napalm modifier. Applied by Zo'rane Fire
+//Increases damage dealt by burn sources
+/datum/modifier/napalm
+	var/burn_mod
+	var/delta
+
+/datum/modifier/napalm/activate()
+	..()
+	delta = strength
+	if (isliving(target))
+		var/mob/living/L = target
+		L.burn_mod *= delta
+
+/datum/modifier/napalm/deactivate()
+	..()
+	if (isliving(target))
+		var/mob/living/L = target
+		L.burn_mod /= delta
```

Comment:
This is bad. Consider a situation where you _add_ to a burn mod afterwards. Basically, this presumes that nothing else touches burn_mod. Which is not necessarily true. Save the added delta instead:
```dm
/datum/modifier/napalm/activate()
	..()
	delta = strength
	if (isliving(target))
		var/mob/living/L = target
		added_burn_mod = L.burn_mod * delta - L.burn_mod
		L.burn_mod += added_burn_mod

/datum/modifier/napalm/deactivate()
	..()
	if (isliving(target))
		var/mob/living/L = target
		L.burn_mod -= added_burn_mod
```